src/Sylius/Behat/Context/Api/Shop/OrderContext.php

```diff
@@ -48,6 +48,29 @@ public function __construct(
     ) {
     }
 
+    /**
+     * @Given I want to change this order's payment method
+     */
+    public function iWantToChangeThisOrdersPaymentMethod(): void
+    {
+        /** @var OrderInterface $order */
+        $order = $this->sharedStorage->get('order');
```

Comment:
```suggestion
     * @Given /^I want to change (this order)'s payment method$/
     */
    public function iWantToChangeThisOrdersPaymentMethod(OrderInterface $order): void
    {
```

Reply:
On the other hand, I'd be for leaving this method completely empty, as it has no sense in the API context and moving all this logic to the `iTryToChangeMyPaymentMethodTo` instead. Then we'll have the whole operation in one place and have no need for:
```php
$this->sharedStorage->set('payment_method_patch_request', $request);
```
You could then most likely reuse the `@When I change my payment method to :paymentMethod` step